lyndapham/step/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,10 +43,29 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Gson gson = new Gson();
+    // String json = gson.toJson(pictureVotes);
```

Comment:
Comments should not be code. Did you mean to delete this?

Reply:
Yes I did! Thanks for catching that.